.readthedocs.yaml

```
@@ -0,0 +1,15 @@
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# RTD API version
+version: 2
+
+build:
+  os: ubuntu-22.04
+  tools:
+    python: >-
+      3.11
+  commands:
+    - python -m venv "${READTHEDOCS_VIRTUALENV_PATH}"
+    - >-
+      "${READTHEDOCS_VIRTUALENV_PATH}"/bin/python deploy.py --output="$READTHEDOCS_OUTPUT/html/"
```

Since you're using `>-` already, why not make the lines shorter?
```suggestion
      "${READTHEDOCS_VIRTUALENV_PATH}"/bin/python deploy.py
      --output="$READTHEDOCS_OUTPUT/html/"
```